.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,76 @@
+---
+name: Bug Report
+description: Create a report to help us improve
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please make sure to fill out the entire form below, providing as much context as you can in order to help us triage and track down your bug as quickly as possible.
+
+        Before filing a bug, please be sure you have searched through [existing bugs](https://github.com/open-telemetry/opentelemetry-python/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug) to see if your bug is already addressed.
+        If your bug is related to an instrumentation or plugin in [opentelemetry-python-contrib](https://github.com/open-telemetry/opentelemetry-python-contrib) please be sure to file it there.
+  
+  - type: textarea
+    id: environment
+    attributes:
+      label: Describe your environment
+      description: |
+        Please describe any aspect of your environment relevant to the problem, including your Python version, [platform](https://docs.python.org/3/library/platform.html), version numbers of installed dependencies, information about your cloud hosting provider, etc. If you're reporting a problem with a specific version of a library in this repo, please check whether the problem has been fixed on main.
+      value: |
+        OS: (e.g, Ubuntu)
+        Python version: (e.g., Python 3.8.10)
+        SDK version: (e.g., )
```

Comment:
Instead of this, what do you think about suggesting that the reporter run `pip freeze` and report the output here?

Reply:
Do you mean suggest it in the `description` or in the `value` block?

Reply:
Probably both. But taking a look at how this renders, I think the output `pip freeze` is important enough to have its own text box. What do you think?

Reply:
I think maybe a section called `Package Version`? Simmillar to this one https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/.github/ISSUE_TEMPLATE/bug_report.yml?plain=1#L36 . But I think we'll have too many fields as well. Can we just use the `environment section` description to instruct reporter to put the package versions? Maybe `Describe your environment and dependencies`.

Reply:
pip freeze may contain a lot of unrelated stuff though and so take a lot of space for not much value.

Reply:
so, can we agree to use the environment section for otel packages version as well?

Reply:
Sure, that's fine with me. I'm partly thinking about users who may not be technically oriented or may not be Python developers, in which case a suggestion like taking the output of `pip freeze` or `pip freeze | grep opentelemetry` could be helpful for filling out these form fields. We OTel developers know what API version and SDK version mean, but a new user (especially one encountering an issue) may not even know that there's a distinction -- or they may not know how to get these values easily enough to not get discouraged during the bug submission process.

I'm also thinking about contributors being able to reproduce the issue reliably and with minimal effort. Maybe the `pip freeze` suggestion could be for the repro section of the form.

Just some thoughts -- not blocking at all, and we can always make adjustments later.